<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1204</width>
    <height>672</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ext Serial Port</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="textEdit_Status">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>361</width>
      <height>551</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>121</width>
      <height>118</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Port</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_Interface"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_open">
       <property name="text">
        <string>Open</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_close">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>40</y>
      <width>381</width>
      <height>551</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>PARAMETER</string>
     </attribute>
     <widget class="QPushButton" name="parameterButton_send">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>440</y>
        <width>291</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Send Parameters</string>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>301</width>
        <height>341</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QComboBox" name="combobox_p">
         <item>
          <property name="text">
           <string>Patient1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Patient2</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QToolBox" name="toolBox">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="page">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>299</width>
            <height>246</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Thresholds</string>
          </attribute>
          <widget class="QWidget" name="gridLayoutWidget">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>266</width>
             <height>146</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="3" column="0">
             <widget class="QCheckBox" name="T3">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>freezeband</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QCheckBox" name="T6">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Dominant F</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QCheckBox" name="T7">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Smoothness</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QCheckBox" name="T2">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>locomotion</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QCheckBox" name="T0">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Interval</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QCheckBox" name="T5">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>FI</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QCheckBox" name="T1">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>auto-correlation</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QCheckBox" name="T4">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Hband</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QCheckBox" name="T8">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Portion</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QWidget" name="page_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>299</width>
            <height>246</height>
           </rect>
          </property>
          <attribute name="label">
           <string>LDA</string>
          </attribute>
          <widget class="QWidget" name="verticalLayoutWidget_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>301</width>
             <height>211</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QComboBox" name="combobox_c">
              <item>
               <property name="text">
                <string>Classifier1</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Classifier2</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <layout class="QGridLayout" name="gridLayout_3">
              <item row="1" column="0">
               <widget class="QCheckBox" name="T1_3">
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>auto-correlation</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QCheckBox" name="T6_3">
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Dominant F</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QCheckBox" name="T3_3">
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>freezeband</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QCheckBox" name="T0_3">
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Interval</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QCheckBox" name="T2_3">
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>locomotion</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QCheckBox" name="T8_3">
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Portion</string>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QCheckBox" name="T4_3">
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Hband</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QCheckBox" name="T7_3">
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Smoothness</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QCheckBox" name="T5_3">
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>FI</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="parameterButton_send_2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>400</y>
        <width>291</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Read Parameters</string>
      </property>
     </widget>
     <widget class="QPushButton" name="training_data">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>480</y>
        <width>121</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>TRAIN</string>
      </property>
     </widget>
     <widget class="QPushButton" name="dataButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>480</y>
        <width>121</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Send D</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>DATA</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>DEBUG INFO</string>
    </property>
   </widget>
   <widget class="QLabel" name="patient">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>70</y>
      <width>221</width>
      <height>511</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/new/prefix1/patient.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="ankle_button">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>520</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="knee_button">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>440</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="waist_button">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>350</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1204</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
